pkg/safeguards/helpers.go

```diff
@@ -127,9 +127,55 @@ func GetManifestFiles(p string) ([]c.ManifestFile, error) {
 
 		if !info.IsDir() && info.Name() != "" && IsYAML(walkPath) {
 			log.Debugf("%s is not a directory, appending to manifestFiles", info.Name())
+			byteContent, err := os.ReadFile(p)
+			if err != nil {
+				return fmt.Errorf("could not read file %s: %s", p, err)
+			}
+			manifest.Name = info.Name()
+			manifest.ManifestContent = byteContent
+			manifestFiles = append(manifestFiles, manifest)
+		} else if !IsYAML(p) {
+			log.Debugf("%s is not a manifest file, skipping...", info.Name())
+		} else {
+			log.Debugf("%s is a directory, skipping...", info.Name())
+		}
 
+		return nil
+	})
+	if err != nil {
+		return nil, fmt.Errorf("could not walk directory: %s", err)
+	}
+	if len(manifestFiles) == 0 {
+		return nil, fmt.Errorf(" found within given path")
```

Comment:
should probably be "no manifest files found within given path"?

Reply:
Good catch, thank you!